src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's contacts in the address book.
+ */
+public class Contact {
+    protected String value;
+    protected boolean isPrivate;
+    
+    @Override
+    public String toString() {
+        return value;
+    }
+    
+    @Override
+    public int hashCode() {
+        return value.hashCode();
+    }
+    
+    public boolean isPrivate() {
+        return isPrivate;
+    }
+}
```

Comment:
i see that you haven't created a constructor in the parent class, think about how you can push more stuff to the parent class in this case and in general.